.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+**IMPORTANT: Please do not create a Pull Request without creating an issue first.**
+
+*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*
+
+Please provide enough information so that others can review your pull request:
+
+<!-- You can skip this if you're fixing a typo -->
+
+Explain the **details** for making this change. What existing problem does the pull request solve?
+
+<!-- Example: When "Adding a function to do X", explain why it is necessary to have a way to do X. -->
+
+**Testing**
+
+<!-- Please test your changes on a clean server prior to the PR - provide screen shots if the code changes outputs or logs.-->
+<!-- Make sure tests pass on both Travis and Circle CI. -->
+
+<!-- **Code formatting** -->
+<!-- Make sure tests pass on Codacy -->
+**Closing issues**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-emphasis-as-heading] Don’t use emphasis to introduce a section, use a heading](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5470889)